python/Scripts/mxvalidate.py

```diff
@@ -13,63 +13,78 @@ def main():
     parser.add_argument("--resolve", dest="resolve", action="store_true", help="Resolve inheritance and string substitutions.")
     parser.add_argument("--verbose", dest="verbose", action="store_true", help="Print summary of elements found in the document.")
     parser.add_argument("--stdlib", dest="stdlib", action="store_true", help="Import standard MaterialX libraries into the document.")
-    parser.add_argument(dest="inputFilename", help="Filename of the input document.")
```

Comment:
Although it's worthwhile to consider extending our `mxvalidate.py` script from individual documents to folders, this seems distinct from the current proposal to make input connections more robust, and I'd recommend omitting it here.

Reply:
Sounds good. I can open a different PR.